src/scss/components/_cards.scss

```diff
@@ -62,5 +66,68 @@ $card-borderColor               : $grey--lightest !default;
             background: $card-bgColor--disabled;
         }
 
+        // creates a section which is separated off by a border on larger screens
+        .c-card-section {
+            padding: 0 spacing(x3);
+            width: calc(100% + #{$card-section-margin});
+
+            @include media('>=mid') {
+                border-top: solid 1px $card-borderColor;
+                width: calc(100% + #{$card-section-margin--aboveMid});
+            }
+        }
+
+            .c-card-section--collapsible {
+                padding: 0 $card-section-collapsible-paddingRight 0 spacing(x2);
+                position: relative;
+                user-select: none;
+
+                @include media('>=mid') {
+                    padding-left: spacing(x3);
+                    padding-right: $card-section-collapsible-paddingRight--aboveMid;
+                }
+
+                &:hover {
+                    cursor: pointer;
+                }
+            }
+
+            .c-card-section--collapsed {
+                height: 50px;
+                overflow: hidden;
+
+                &.c-card-section {
+                    margin-bottom: -#{spacing(x2)};
+                }
+
+                p {
```

Comment:
Should add a class and target that instead, something like `.c-card-section-text` - just incase the the `p` changes in the future :)

Reply:
The thing is, the text comes from the back-end somewhere so we can't easily add a class to the content.

Reply:
ahh - bummer. No worries then 😄

Reply:
I'll keep your comment in mind though as there may be an opportunity to fix that text from within the menu vertical.